scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -70,16 +72,22 @@
 1. `GvsCallsetStatistics` workflow
-    - You will need to run `GvsPrepareRangesCallset` workflow first, if it has not been run already
-       - This workflow transforms the data in the vet tables into a schema optimized for callset stats creation and for calculating sensitivity and precision.
-       - The `only_output_vet_tables` input should be set to `true` (the default value is `false`).
+    - You will need to run `GvsPrepareRangesCallset` workflow first, if it has not been run already. Note that running this workflow is also a prerequisite for running the `GvsExtractCallsetPgenMerged` workflow discussed below.
+       - This workflow transforms the data in the vet, ref_ranges, and sample_info tables into a schema optimized for callset stats creation and for calculating sensitivity and precision, as well as for PGEN extract.
        - The `enable_extract_table_ttl` input should be set to `true` (the default value is `false`), which will add a TTL of two weeks to the tables it creates.
        - See [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the callset stats.
        - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
     - You will need to have the "BigQuery Data Viewer" role for your @pmi-ops proxy group on the `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites` table
     - This workflow needs to be run with the `extract_table_prefix` input from `GvsPrepareRangesCallset` step.
     - This workflow needs to be run with the `filter_set_name` input from `GvsCreateFilterSet` step.
     - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+1. `GvsExtractCallsetPgenMerged` workflow
+    - Run the `GvsPrepareRangesCallset` workflow first following the steps described above for the `GvsCallsetStatistics` workflow.
+    - Specify the same `call_set_identifier`, `dataset_name`, `project_id`, and `extract_table_prefix` that were used in the `GvsPrepareRangesCallset` run documented above.
+    - Specify the `interval_list` and `interval_weights_bed` appropriate for the PGEN extraction run you are performing. `gs://gvs_quickstart_storage/weights/gvs_full_vet_weights_1kb_padded_orig.bed` is the interval weights BED used for Quickstart. The expectation is that there will be multiple PGEN extraction runs for a variety of interval lists:
+      - [Exome-only](https://broadworkbench.atlassian.net/browse/VS-1279)
+      - [ACAF threshold](https://broadworkbench.atlassian.net/browse/VS-1109)
+      - others?
```

Comment:
I'm kinda uncomfortable to adding callset-specific tickets to a "general" doc for AoU callsets.

Reply:
Happy to remove this as I'm struggling to get documentation on what interval lists we're going to use for PGEN / VCF extract anyway...

Reply:
You can see that the three deliverables are "ACAF Threshold", "Exome" and "ClinVar" here: https://support.researchallofus.org/hc/en-us/articles/14929793660948-Smaller-Callsets-for-Analyzing-Short-Read-WGS-SNP-Indel-Data-with-Hail-MT-VCF-and-PLINK

Reply:
Sweet, thank you!